lib/ansible/modules/cron.py

```diff
@@ -131,6 +131,7 @@
     version_added: "2.1"
 requirements:
   - cron (any 'vixie cron' conformant variant, like cronie)
+  - if used for newer MacOS targets, first enable these options under "Privacy" -> "Full Disk Access" - C(ssd-keygen-wrapper), C(Terminal) and C(cron).
```

Comment:
```suggestion
notes:
  - If cron has permissions issues in the latest versions of MacOS, you will need to grant cron full disk access on the Mac, please refer to the official documentation for more details.
```

Reply:
Or maybe better like this?

"...you will need to grant cron, ssd-keygen-wrapper and terminal full disk access, please refer..."

Cause it's not enough to grant the access to cron only to enable this module to do its thing.